tools/scripts/stm32.mk

```diff
@@ -96,6 +147,39 @@ $(PLATFORM)_project:
 	$(call remove_file,$(BINARY).cubemx) $(HIDE)
 	$(MAKE) --no-print-directory $(PROJECT)_configure
 
+endif #STM32_EXT_BUILD
+
+ifeq ($(STM32_EXT_BUILD),y)
+$(PROJECT)_configure:
```

Comment:
this whole _configure sequence shares a lot with the original sequence below, i don't think it's a good idea to duplicate it like this